src/components/header/help-modal/index.tsx

```diff
@@ -67,8 +67,12 @@ const HelpModal = () => {
       </a>
       <h2>Tip</h2>
       <p>
-        You can access Vega, Vega-Lite, and the <a href="https://vega.github.io/vega/docs/api/view/"> Vega view</a> at{' '}
-        <code>(VEGA_DEBUG)</code> using your browser's developer console.
+        You can access Vega, Vega-Lite, and the{' '}
```

Comment:
Do we need space before and after the opening tag?

Reply:
Yeah , that is for some space between the <code> and the 'the'.

Reply:
I am opening a new issue to improve the css of the help modal

Reply:
yes